client/asset/dcr/dcr.go

```diff
@@ -5707,7 +5814,7 @@ func (dcr *ExchangeWallet) checkPendingTxs(ctx context.Context, tip uint64) {
 
 		if lastQuery != 0 && lastQuery < tip-blockQueryBuffer {
 			blockToQuery = lastQuery - blockQueryBuffer
-		} else if lastQuery >= tip-blockQueryBuffer {
+		} else {
 			blockToQuery = tip - blockQueryBuffer
 		}
 
```

Comment:
When I connect the beta wallet, I don't see any transactions. I dug in a little bit, and I see that the first `lastQuery` is `0`, which because of the `lastQuery != 0` condition gives us only the `tip - blockQueryBuffer` blocks to scan. Shouldn't it scan from 0?